pdns/comfun.cc

```diff
@@ -69,8 +69,9 @@ struct SendReceive
   {
     d_socket = socket(AF_INET, SOCK_DGRAM, 0);
     int val=1;
-    setsockopt(d_socket, SOL_SOCKET, SO_REUSEADDR, &val, sizeof(val));
-    
+    if (setsockopt(d_socket, SOL_SOCKET, SO_REUSEADDR, &val, sizeof(val)) != 0)
+       g_log<<Logger::Warning<<"Failed to turn on SO_REUSEADDR: " << stringerror() << endl;
```

Comment:
Are yo sure this is not going to spam the logs if it fails. Is it really useful to know?